roaring/containers.go

```diff
@@ -319,23 +319,24 @@ func (sc *sliceContainers) Repair() {
 }
 
 type sliceIterator struct {
-	e     *sliceContainers
-	i     int
-	key   uint64
-	value *Container
+	e *sliceContainers
+	i int
 }
 
-func (si *sliceIterator) Next() bool {
+func (si sliceIterator) Next() bool {
 	if si.e == nil || si.i > len(si.e.keys)-1 {
 		return false
 	}
-	si.key = si.e.keys[si.i]
-	si.value = si.e.containers[si.i]
-	si.i++
 
+	si.i++
```

Comment:
Not sure how this is intended to be used, but you probably need this to be a method on `*sliceIterator` for this to take effect?

Reply:
yeah I pushed the wrong version, should be good now

Reply:
how did you even find this p.r lol

Reply:
I auto subscribe to new repos =]